sensor_msgs/Image img
geometry_msgs/Point[] edges_2d